frontend/pages/SoftwarePage/SoftwareTitles/SoftwareTable/SoftwareTitlesTableConfig.tsx

```diff
@@ -91,16 +92,21 @@ const generateTableHeaders = (
       Cell: (cellProps: IStringCellProps): JSX.Element => {
         const { id, name, source } = cellProps.row.original;
 
+        const teamQueryParam = buildQueryStringFromParams({ team_id: teamId });
```

Comment:
@jacobshandling added the team ID to the name column here

Reply:
You also need it in `OperatingSystemsTableConfig`